graphql/queries.ts

```diff
@@ -99,10 +110,7 @@ export const GET_APP = gql`
         order
         challengeId
         lessonId
-        reviewer {
-          id
-          username
```

Comment:
don't you want to view who reviewed your submission? I think reviewer id and username is important.

Reply:
good point haha